docs/guides/connecting_to_your_data/database/snowflake.md

```diff
@@ -1,4 +1,162 @@
+import WhereToRunCode from '../components/where_to_run_code.md'
+import NextSteps from '../components/next_steps.md'
+import Congratulations from '../components/congratulations.md'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This guide will help you connect to data in a Snowflake database.
+This will allow you to validate and explore your data.
+
+<Prerequisites>
+
+- Have access to data in a Snowflake database
+
+</Prerequisites>
+
+## Steps
+
+### 1. Choose how to run the code in this guide
+
+<WhereToRunCode />
+
+### 2. Install required dependencies
+
+First, install the necessary dependencies for Great Expectations to connect to your Snowflake database by running the following in your terminal:
+
+```console
+pip install sqlalchemy
+pip install snowflake-connector-python
+pip install snowflake-sqlalchemy
+```
+
+### 3. Add credentials
+
+Great Expectations provides multiple methods of using credentials for accessing databases.
+Options include using an file not checked into source control, environment variables, and using a cloud secret store.
+Please read the article [Credential storage and usage options](../advanced/database_credentials) for instructions on alternatives.
+
+For this guide we will use a `connection_string` like this:
+
+```
+snowflake://<USER_NAME>:<PASSWORD>@<ACCOUNT_NAME>/<DATABASE_NAME>/<SCHEMA_NAME>?warehouse=<WAREHOUSE_NAME>&role=<ROLE_NAME>
+```   
+
+### 4. `[🍏 CORE SKILL ICON]` Instantiate your project's DataContext
+
+Import these necessary packages and modules.
+
+```python file=../../../../integration/code/connecting_to_your_data/database/snowflake_yaml_example.py#L1-L3
+```
+
+Load your DataContext into memory using the `get_context()` method.
+
+```python file=../../../../integration/code/connecting_to_your_data/database/snowflake_yaml_example.py#L20
+```
+
+### 5. Configure your Datasource
+
+<Tabs
+  groupId="yaml-or-python"
+  defaultValue='yaml'
+  values={[
+  {label: 'YAML', value:'yaml'},
+  {label: 'python', value:'python'},
+  ]}>
+  <TabItem value="yaml">
+
+Put your connection string in this template:
+
+```python file=../../../../integration/code/connecting_to_your_data/database/snowflake_yaml_example.py#L22-L36
+```
+Run this code to test your configuration.
+```python file=../../../../integration/code/connecting_to_your_data/database/snowflake_yaml_example.py#L43
+```
+
+</TabItem>
+<TabItem value="python">
+
+Put your connection string in this template:
+
+```python file=../../../../integration/code/connecting_to_your_data/database/snowflake_python_example.py#L23-L40
+```
+Run this code to test your configuration.
+```python file=../../../../integration/code/connecting_to_your_data/database/snowflake_python_example.py#L44
+```
+
+</TabItem>
+</Tabs>
+
+You will see your database tables listed as `Available data_asset_names` in the output.
+
+Feel free to adjust your configuration and re-run `test_yaml_config` as needed.
```

Comment:
Please update these two sentences to match the ones now in `knoxpod` after your feedback about test yaml config. 😄

Reply:
Will do :)